toolkit/scss/forms/_questions.scss

```diff
@@ -12,6 +12,11 @@
   .hint {
     margin-bottom: 0;
   }
+
+  // This preserves the vertical margins set on the .question-headings
+  legend {
```

Comment:
Yeah, sure. Good point.

The bottom margin is already there but it's getting ignored, which's means that we should either
i) add this so that it shows up, or
ii) remove the margin

Having the rule and not using it seems like the wrong thing to be doing.